terraform/deployments/github/main.tf

```
@@ -56,11 +56,21 @@ data "github_repository" "govuk" {
   full_name = each.key
 }
 
+data "github_repository" "govuk_repo_names" {
+  for_each = toset(data.github_repositories.govuk.names)
+  name     = each.key
+}
+
 locals {
   deployable_repos = [
     for r in data.github_repository.govuk : r
     if !r.fork && contains(r.topics, "container")
   ]
+
+  permission_repos = [
```

I'd suggest a slightly more expressive name, e.g.

```suggestion
  auto_configurable_repos = [
```

Repos marked 'sensitive access' are not auto configurable, they need managing carefully & manually. So I think that makes more sense as a name.